### PR TITLE
Cart Page

### PR DESCRIPTION
**Pull request by bharati-21**
Created cart page
- Contains items in the cart with price and quantity
- Remove from cart, add to wishlist buttons
- Container that displays the total price of all items in the cart
- Place order button

PREVIEW
image

### CODE REVIEW

File: styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.cart-main-header {
+    border-bottom: 2px solid var(--gray);
+    grid-column: 1 / -1;
+}
+
+.shelf-products {
+    width: 100%;
+    --bg-horizontal-card: #F0f0f0;
```
Comment (yashpatel1797) [inline styles/cart.css:13]: use var here also.

File: styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+@media screen and (max-width: 460px) {
+    .shelf-product-card .card-main {
+        flex-direction: column;
+    }
+
+    .shelf-product-card .card-body {
+        padding-left: 0;
+        padding-right: 0;
```
Comment (yashpatel1797) [inline styles/cart.css:132]: use Shorthand properties here.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Review by AtulHande03a — APPROVED** [review]
looks good to merge

File: styles/cart.css (hunk 1 above)
Comment (bharati-21) [reply styles/cart.css:13]: Actually I'm setting a variable here locally to the shelf products.

File: styles/cart.css (hunk 2 above)
Comment (bharati-21) [reply styles/cart.css:132]: Yes, thank you.

**Comment by bharati-21** [comment]
> looks good to merge

Thank you

**Pull request merged**